BridgeAppSDK/SBARoundedButton.swift

```diff
@@ -85,6 +117,11 @@ import UIKit
     }
     
     func refreshView() {
+        guard self.alpha > 0.9 else {
```

Comment:
It's there as a "soft marker" for indicating that the view is "hidden". ResearchKit has different places where it hides the continue button by setting alpha to `0` rather than using `isHidden`. This is a work-around to allow for a consistent continue button.